src/main/kotlin/com/hibob/academy/resource/PetResource.kt

```diff
@@ -22,8 +22,8 @@ class PetResource(
     @GET
     fun getALLPets() : Response{
         val pets = listOf(
-            Pet(1L, "Bobby", PetType.DOG, LocalDate.parse("2021-01-01"), 1),
-            Pet(2L, "Kitty", PetType.CAT, LocalDate.parse("2021-01-01"), 1)
+            Pet(1L, "Bobby", PetType.DOG, LocalDate.parse("2021-01-01"), 1, 1L),
```

Comment:
No need to implement yet or it was missed?

Reply:
No need to implement it yet. but I can write it